src/som/interpreter/actors/Actor.java

```diff
@@ -75,6 +77,9 @@ public static Actor createActor(final VM vm) {
   /** Is scheduled on the pool, and executes messages to this actor. */
   protected final ExecAllMessages executor;
 
+  /** Flag that indicates if a step-to-next-turn action has been made in the previous message. */
+  protected boolean stepToNextTurn;
```

Comment:
Tracing is not optional, and it hasn't been optional for quite some time already (4-5month).
We need tracing data in the debugger for the visualization as well as the code panels.
The code that shows that is not optional is here:
https://github.com/smarr/SOMns/blob/master/src/som/vm/VmSettings.java#L58